op-supervisor/supervisor/backend/cross/cycle_test.go

```diff
@@ -489,6 +488,8 @@ func TestHazardCycleChecksCycle(t *testing.T) {
 			msg:       "expected cycle detection error for 3-node cycle",
 		},
 		{
+			// 1->2->3->4->5->1
+			// 1->2->5->1
```

Comment:
```suggestion
			// 1->5->4->3->2->1  - executing message, and multiple regular log ordering steps
			// 1->5->2->1 - two executing messages and single regular log ordering dependency
```
The arrows seemed the other way around, compared to the other test case descriptions: `1` depends on `5`, and `5` then has prior logs. (assuming we are using `exec -> init` as format).

And the `5` depending on `2` part in the test is valid, not a cycle without the other 1->5 link.